### PR TITLE
Adds player inventory interaction / stripping

### LINKED ISSUE #2144: Inventory inspection, Cloth stripping/stealing functionality
# Description
Would be nice to be able to see a list of other person's equipment with ability to take it off,
because at the moment there's no way to retrieve dead person's belongings

### PR DESCRIPTION
**Pull request by chairbender**
### Purpose
Fixes #2144 

Includes #2279 PR, hopefully this time it will actually recognize the commits are the same when that is merged. Sorry it's so soon after, I didn't realized I'd have this ready this quickly.

Can click drag a non-conscious player to yourself to view their inventory. A button allows dropping all of their items on the ground, or you can interact with their inventory slots just like it was a storage you are looking into.

It's a bit "quick and dirty" as far as the UI goes but at least now you can do it and items aren't lost forever in their inventory.

### Open Questions and Pre-Merge TODOs

- [X]  This fix is tested on the same branch it is PR'ed to.
- [X]  I correctly commented my code
- [X]  My code is indented with tabs and not spaces
- [X]  This PR does not include any unnecessary .meta, .prefab or .unity (scene) changes
- [X]  This PR does not bring up any new compile errors
- [X]  This PR has been tested in editor
- [X]  Any new / changed components have been [tested using Respawn](https://github.com/unitystation/unitystation/wiki/Proper-Spawning-and-Despawning-%28Object-Lifecycle%29)
- [X]  This PR has been tested with round restarts.
- [X]  This PR has been tested on moving / rotating / rotated-before-joining matrices (if applicable)
- [X]  This PR has been tested in multiplayer (with 2 clients and late joiners, if applicable)

### CODE REVIEW

**Comment by chairbender** [comment]
Also the "Networking > Respawn Player" command should work now regardless of game mode.

File: UnityProject/Assets/Scripts/Clothing/BackPack/InteractableStorage.cs [hunk 1]
```diff
@@ -92,7 +94,15 @@ public bool WillInteract(MouseDrop interaction, NetworkSide side)
 		else
 		{
 			//trying to view this storage, can only drop on ourselves to view it
-			return interaction.Performer == interaction.TargetObject;
+			if (interaction.Performer != interaction.TargetObject) return false;
+			//if we're dragging another player to us, it's only allowed if the other player is downed
+			if (Validations.HasComponent<PlayerScript>(interaction.DroppedObject))
+			{
+				//dragging a player, can only do this if they are down / dead
+				return Validations.IsDeadOrCrit(interaction.DroppedObject, side);
```
Comment (fomalsd) [inline UnityProject/Assets/Scripts/Clothing/BackPack/InteractableStorage.cs:102]: how about also checking for "stunned" (`registerPlayer.IsSlippingServer`) 
and "cuffed" (apparently `playerMove.IsCuffed`)?

**Review by fomalsd — COMMENTED** [review]
I'm afraid you'll have to exclude tab message split code changes from this one due to its unreliability

**Comment by chairbender** [comment]
need to redo this PR to remove the ID console stuff, closing.

**Pull request closed without merging**